feat(pictograms): adds pictos and updates categories

chrisconnors-ibm opened the pull request:
#### Changelog

**New**

- mexico--city--angel--of--independence.svg
- mexico--city--museo--soumaya.svg
- milan--duomo--di--milano.svg
- milan--skyscrapers.svg


**Changed**

- categories.yml
- pictogram-master.ai
- metadata.yml

**reviewers**

verify svgs exist/work. search against aliases (`museum`and `statue`). all four grouped within `Cities`.

chrisconnors-ibm committed:
adds pictos and updates categories

adds:
mexico--city--angel--of--independence.svg
mexico--city--museo--soumaya.svg
milan--duomo--di--milano.svg
milan--skyscrapers.svg

updates
categories.yml

jnm2377 commented on packages/pictograms/categories.yml:
```diff
@@ -61,6 +61,10 @@ categories:
+    - mexico--city--angel--of--independence
+    - mexico--city--museo--soumaya
+    - milan--duomo--di--milano
+    - milan--skyscrapers.svg
```
Is there a reason this one includes `.svg` and the others don't?

joshblack replied:
```suggestion
    - milan--skyscrapers
```

per @jnm2377 comments 👍

jnm2377 commented:
Also can we update the commit title to use proper commit syntax? Thanks!

chrisconnors-ibm committed:
Update packages/pictograms/categories.yml

sorry, no PRs in haste!

Co-Authored-By: Josh Black <josh@josh.black>

jnm2377 commented:
hmm... I think the pictograms need to be added to testing maybe? It seems like ci is failing bc it says they're missing.

jnm2377 committed:
Merge branch 'master' into picto-update

joshblack reviewed (changes requested):
Looks like CI is noting that they need to be defined in metadata.yml

joshblack commented:
Bump @chrisconnors-ibm do you have a sec to add in the metadata into `metadata.yml`?

joshblack committed:
Merge branch 'master' into picto-update

chrisconnors-ibm committed:
Update metadata.yml

add items to metadata.yml

chrisconnors-ibm committed:
Update metadata.yml

corrected the new entries

joshblack committed:
chore(pictograms): update metadata

joshblack committed:
Merge branch 'master' into picto-update

joshblack committed:
Merge branch 'master' into picto-update

chrisconnors-ibm committed:
mv `mexico--city*` `mexico-city*`

change delimiter in `mexico--city` to `mexico-city` filenames and categoreis.yml and metadata.yml

vpicone committed:
Merge branch 'master' into picto-update
